src/filters.js

```diff
@@ -84,7 +84,7 @@ const makeBetterSafe = ({ markAsSafe }) => {
 const makeBuildStaticAddonCards =
   ({ _buildStaticAddonCard = buildStaticAddonCard } = {}) =>
   async (value, callback) => {
-    const regexp = /<div class="addon-card" data-addon-id="(\d+)"><\/div>/g;
+    const regexp = /<div class="addon-card" data-addon-id="(\S+)"><\/div>/g;
```

Comment:
I looked at my notifications in the wrong order so I left a comment in the other PR: https://github.com/mozilla/addons-blog/pull/344#discussion_r759661089

TL;DR: I'd like us to avoid using `\S+` because it is almost like `\s` and that's not very readable. Instead, I'd suggest `.+?` to match everything inside the quotes.